### ISSUE TITLE
plot_filters_1d has the wrong images

### ISSUE THREAD

**Issue by janden** [kymatio/kymatio#812]
When the HTML is generated, we get the images from plot_filters_2d instead of plot_filters_1d: https://www.kymat.io/gallery_1d/plot_filters.html

Probably some images are getting overwritten when the docs are built.

**Comment by rastegah** [comment]
indeed. 

from what I remember these tutorials do not work with the intro of stride on `dev`

**Comment by rastegah** [comment]
fixed by this PR https://github.com/kymatio/kymatio/pull/815

**Comment by janden** [comment]
Sorry to say this but it's not working for me. I get

```
$ python3 plot_filters.py 
Traceback (most recent call last):
  File "/home/janden/projects/kymatio/examples/1d/plot_filters.py", line 62, in <module>
    phi_f, psi1_f, psi2_f = scattering_filter_factory(np.log2(T), J, Q, T)
  File "/home/janden/projects/kymatio/kymatio/scattering1d/filter_bank.py", line 446, in scattering_filter_factory
    psi_levels.append(periodize_filter_fourier(psi_levels[0], nperiods))
  File "/home/janden/projects/kymatio/kymatio/scattering1d/filter_bank.py", line 70, in periodize_filter_fourier
    v_f = h_f.reshape(nperiods, N).mean(axis=0)
ValueError: cannot reshape array of size 13 into shape (2,6)
```

**Comment by lostanlen** [comment]
maybe some https://github.com/kymatio/kymatio/pull/863 backfire ?

**Comment by lostanlen** [comment]
fixed by #899
thanks again @/janden
